packages/discovery-provider/src/tasks/entity_manager/entities/user.py

```diff
@@ -611,7 +620,32 @@ def verify_user(params: ManageEntityParameters):
     )
 
     user_record = validate_user_record(user_record)
-    user_record.is_verified = True
+
+    metadata = params.metadata
+    # todo: default below to false instead of true after identity change has been deployed
+    is_verified = metadata.get("is_verified", True)
```

Comment:
is this vulnerable to abuse if metadata leaves out `is_verified` and this defaults to true?

Reply:
sync'd. but this is for backwards compatibility and its only in verify flow which is separately verified to make sure its from the identity service.